src/main/resources/hudson/plugins/build_timeout/operations/AbortAndRestartOperation/config_ja.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Likely\ stuck=\u6ede\u7559\u3057\u3066\u3044\u308b\u53ef\u80fd\u6027\u304c\u9ad8\u3044\u306a\u3089\u3070\u4e2d\u6b62
+Maximum\ restarts=\u6700\u5927\u518d\u8d77\u52d5\u56de\u6570
```

Comment:
Why did you remove LikelyStuckTimeOutStrategy/config_ja.properties?
(and why did you committed it as renaming?)